### PR TITLE
Relics of the Past Encounter Cards

### PR DESCRIPTION
**Pull request by somethingtemporaryarkham**
Adding the rest of the encounter cards for this scenario

### CODE REVIEW

**Comment by somethingtemporaryarkham** [comment]
@fspoettel @drawntotheflame Pinging for review!

File: pack/parallel/rop_encounter.json [hunk 1]
```diff
@@ -12,5 +12,242 @@
...
+    "encounter_position": 11,
+    "faction_code": "neutral",
+    "illustrator": "Rob Laskey",
+    "is_unique": true,
+    "name": "Citrine Snake",
+    "pack_code": "rop",
+    "position": 75,
+    "quantity": 1,
+    "subname": "Effigy of the Child",
+    "text": "<b>Revelation</b> – Put Citrine Snake into play under your control. If you are poisoned, take 1 direct damage. If you are not poisoned, put a set-aside Poisoned weakness into play in your threat area.\n<b>Forced</b> – When you resign: Add Citrine Snake to the victory display.",
```
Comment (fspoettel) [inline pack/parallel/rop_encounter.json:216]:
```suggestion
    "text": "<b>Revelation</b> - Put Citrine Snake into play under your control. If you are poisoned, take 1 direct damage. If you are not poisoned, put a set-aside Poisoned weakness into play in your threat area.\n<b>Forced</b> - When you resign: Add Citrine Snake to the victory display.",
```

File: pack/parallel/rop_encounter.json [hunk 2]
```diff
@@ -12,5 +12,242 @@
...
+    "encounter_position": 12,
+    "faction_code": "neutral",
+    "illustrator": "Rob Laskey",
+    "is_unique": true,
+    "name": "Turquoise Eagle",
+    "pack_code": "rop",
+    "position": 76,
+    "quantity": 1,
+    "subname": "Effigy of the Watcher",
+    "text": "<b>Revelation</b> – Put Turquoise Eagle into play under your control. You must either place 1 doom on your location, or take 2 horror.\n<b>Forced</b> – When you resign: Add Turquoise Eagle to the victory display.",
```
Comment (fspoettel) [inline pack/parallel/rop_encounter.json:234]:
```suggestion
    "text": "<b>Revelation</b> - Put Turquoise Eagle into play under your control. You must either place 1 doom on your location, or take 2 horror.\n<b>Forced</b> - When you resign: Add Turquoise Eagle to the victory display.",
```

File: pack/parallel/rop_encounter.json [hunk 3]
```diff
@@ -12,5 +12,242 @@
...
+    "code": "90077",
+    "encounter_code": "relics_of_the_past",
+    "encounter_position": 13,
+    "faction_code": "mythos",
+    "illustrator": "Rob Laskey",
+    "name": "Vengeant Past",
+    "pack_code": "rop",
+    "position": 77,
+    "quantity": 1,
+    "text": "Peril.\n<b>Revelation</b> – For each [[Ancient]] story asset controlled by an investigator, you must either <i>choose one)</i>:\n- Deal 1 damage or horror to that asset's controller.\n-Shuffle that asset into the exploration deck.\nAfter this effect resolves, shuffle Vengeant Past into the exploration deck.",
```
Comment (fspoettel) [inline pack/parallel/rop_encounter.json:249]:
```suggestion
    "text": "Peril.\n<b>Revelation</b> - For each [[Ancient]] story asset controlled by an investigator, you must either <i>(choose one)</i>:\n- Deal 1 damage or horror to that asset's controller.\n- Shuffle that asset into the exploration deck.\nAfter this effect resolves, shuffle Vengeant Past into the exploration deck.",
```

**Comment by drawntotheflame** [comment]
@Kamalisk this can go live, thanks.

**Comment by morvael** [comment]
Would be so nice to get this into the DB :)

**Pull request merged**